modules/rosa-sts-associating-your-aws-account.adoc

```diff
@@ -0,0 +1,47 @@
+// Module included in the following assemblies:
+//
+// * rosa_install_access_delete_clusters/rosa-sts-creating-a-cluster-quickly.adoc
+
+:_content-type: PROCEDURE
+[id="rosa-sts-associating-your-aws-account_{context}"]
+= Associating your AWS account with your Red Hat organization
+
+Before using {cluster-manager-first} to create {product-title} (ROSA) clusters that use the AWS Security Token Service (STS), create an {cluster-manager} IAM role and link it to your Red Hat organization. Then, create a user IAM role and link it to your Red Hat user account in the same Red Hat organization.
+
+.Prerequisites
+
+* You have completed the AWS prerequisites for ROSA with STS.
+* You have available AWS service quotas.
+* You have enabled the ROSA service in the AWS Console.
+* You have installed and configured the latest ROSA CLI (`rosa`) on your installation host.
++
+[NOTE]
+====
+To successfully install ROSA clusters, use the latest version of the ROSA CLI.
+====
+* You have logged in to your Red Hat account by using the `rosa` CLI.
+
+.Procedure
+
+. Create an {cluster-manager} role and link it to your Red Hat organization:
```

Comment:
@arendej can a non org admin create a ocm role now and link it ? if not @pneedle-rh  we need to mention i think

Reply:
Thank you for highlighting this. Based on the prerequisites in the "Creating an OpenShift Cluster Manager IAM role" subsection in https://docs.openshift.com/rosa/rosa_architecture/rosa-sts-about-iam-resources.html#rosa-sts-understanding-ocm-role_rosa-sts-about-iam-resources, it looks like the org admin privileges are still required.

I have subsequently added the following line to the prerequisites in `modules/rosa-sts-associating-your-aws-account.adoc` in this PR (49293):

> * You have organization administrator privileges in your Red Hat organization.